examples/behavior_writer/export_behavior_to_csv.py

```diff
@@ -2,7 +2,7 @@
 from csv import DictWriter
 
 import click
-from xpanse.client import ExClient
+from xpanse.client import XpanseClient
```

Comment:
We could go ahead and delete this, this looks related to behavior data (which we deprecated a while back)

Reply:
I can delete all the examples since we'll need to redo them anyway